oxide/types.go

```diff
@@ -2609,7 +2635,7 @@ type ExternalIpCreateFloating struct {
 // ExternalIpCreate is parameters for creating an external IP address for instances.
 type ExternalIpCreate struct {
 	// Pool is the type definition for a Pool.
-	Pool NameOrId `json:"pool,omitempty" yaml:"pool,omitempty"`
+	Pool NameOrId `json:"pool,omitzero" yaml:"pool,omitzero"`
```

Comment:
The change in the generation code seems to have changed this as well. Can we run the instance terraform acceptance tests against this, or validate the behaviour has remained the same in some other way? Sorry, I'm so paranoid with omitempty changes in the SDK now 😅